mitodl/ol-data-platform
# create marts for MITx Online certificate and profile data

# What are the relevant tickets?




https://github.com/mitodl/hq/issues/1716
https://github.com/mitodl/hq/issues/1748

# Description (What does it do?)

This adds `marts__mitxonline_course_certificates` and `marts__mitxonline_user_profiles` for IRx. 

# How can this be tested?

Run the following commands against your schema or QA
Run `dbt build --select marts.mitxonline` if you have all intermediate models, otherwise add + in front of marts.mitxonline to run upstream models
Run `dbt test --select int__mitxonline__users` as well as `int__bootcamp__users` and `int__mitxpro__users` to ensure the test pass since I made the change to accepted_values for user_highest_education
